sdk/storage/Azure.Storage.DataMovement.Blobs/src/BlobDirectoryStorageResourceContainer.cs

```diff
@@ -96,10 +96,10 @@ public override StorageResource GetChildStorageResource(string encodedPath)
         }
 
         /// <summary>
-        /// Retrieves a single blob resoruce based on this respective resource.
+        /// Retrieves a single blob resource based on this respective resource.
         /// </summary>
-        /// <param name="encodedPath"></param>
-        /// <param name="type"></param>
+        /// <param name="encodedPath">A URL-encoded path as it would appear in a request URI.</param>
```

Comment:
Want to callout to check if this docstring is accurate (I modeled it after the description of encoded path found in .NET Blobs package)

Reply:
Yeah this is accurate.

Reply:
Nvm, let's just remove URL-encoded. I'm going to rename this back to path in another PR. (see the other comment you made about `path` and `encodedPath`